src/pages/label-template/Windows/LabelTemplateWindow.js

```diff
@@ -0,0 +1,37 @@
+// ** Custom Imports
+import CustomTabPanel from 'src/components/Shared/CustomTabPanel'
+import { useState } from 'react'
+import { CustomTabs } from 'src/components/Shared/CustomTabs'
+import LabelTemplateForm from '../Forms/LabelTemplateForm'
+import ItemList from '../Forms/ItemList'
+
+const LabelTemplateWindow = ({ labels, editMode, maxAccess, recordId, height }) => {
```

Comment:
you are not passing maxAccess & height from index
height if not serving any purpose remove it